### ISSUE TITLE
Specs passing, but no Gist

### ISSUE THREAD

**Issue by onyoo** [learn-co-curriculum/js-apis-lab#4]
I'm passing all the specs, but am not able to create a Gist as intended. The fist student to finish didn't seem to either.

First sign things were wrong was the response didn't seem to match what was expected. Maybe I generated the wrong kind of GH token? If so, maybe there could be a tutorial on how to get the right on? see response object :

<img width="830" alt="screen shot 2016-02-19 at 6 47 57 pm" src="https://cloud.githubusercontent.com/assets/14326883/13183918/679f5880-d739-11e5-9326-acc9559c6559.png">

I'm not sure how to fix this, but 2/2 students had the same problem, so maybe the rest will too.

This was how I created my token:

<img width="799" alt="screen shot 2016-02-19 at 6 51 35 pm" src="https://cloud.githubusercontent.com/assets/14326883/13184023/eaf1450e-d739-11e5-879b-79d14ae736cc.png">

**Comment by victhevenot** [comment]
hey @onyoo if you push your code up to github, I'm happy to take a look!

**Comment by onyoo** [comment]
Sure thing: https://github.com/onyoo/js-apis-lab-v-000/blob/master/js/github_client.js

Thanks for the help!

**Comment by victhevenot** [comment]
hey @onyoo,

After playing around with your solution and Flatiron's, I found the source of the issue. Instead of using jsonp as the dataType of the post request, you want to just use `json`. and that should solve the problem. im going to update the readme to reflect that.

Also, if you want to get a sense of how I found the issue. I made my API token with the same selections as you. Then, I tried to make the POST request using Postman, which was successful. From there, I used the debugger, and looked at the `Network` tab in the developer tools, and notice that there were not request headers in the POST request you made (which contain information for GIthub's server to know how to process my request). 

basically, `jsonp` overwrites the request headers, so you weren't passing your authorization info to Github at all. Once you change it to `json` all should work well!

**Comment by onyoo** [comment]
@vicfriedman Thank you so much for your detailed response! I was having trouble with headers in general in this section. I think headers are covered at the beginning of the intro to the internet lesson, which seems like eons ago. Still, I should have caught this.

I will follow your steps to the letter when I finish my current lab. Thanks again!!
